extensions/aws/s3/S3WrapperBase.cpp

```diff
@@ -68,13 +75,13 @@ void S3WrapperBase::setCannedAcl(Aws::S3::Model::PutObjectRequest& request, cons
   request.SetACL(CANNED_ACL_MAP.at(canned_acl));
 }
 
-std::string S3WrapperBase::getExpiryDate(const std::string& expiration) {
+std::pair<std::string, std::string> S3WrapperBase::getExpirationPair(const std::string& expiration) {
   static const std::regex expr = std::regex("expiry-date=\"(.*)\", rule-id=\"(.*)\"");
```

Comment:
shouldn't we use the `Regex` class defined in `RegexUtils.h`? (that contains some "magic" for `<GCC 4.9`)

Reply:
Good shout

Reply:
Fixed in [8ddbc42](https://github.com/apache/nifi-minifi-cpp/pull/970/commits/8ddbc42c1e49ffcbd7764be5b95535bbf9e65813)